### PR TITLE
Create PrathamRathod.json

### PR DESCRIPTION
**Pull request by PRATHAM-099**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4311"></a>

### CODE REVIEW

File: PrathamRathod.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/pratham_rathod_99/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://twitter.com/_PrathamRathod_",
```
Comment (kumarsonsoff3) [inline PrathamRathod.json:28]: You might want to use your GitHub profile here. Please update it with your GitHub URL. 

```suggestion
      "url": "https://github.com/PRATHAM-099",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please update your GitHub link as suggested below + rename your [file here](https://github.com/PRATHAM-099/LinkFree/edit/PRATHAM-099/PrathamRathod.json) to exact `data/PRATHAM-099.json`. 

Let us know if you need any help with this! :)
image

File: PrathamRathod.json (hunk 1 above)
Comment (PRATHAM-099) [reply PrathamRathod.json]: it's done

**Comment by PRATHAM-099** [comment]
name of the file is changed to data/PRATHAM-099.json

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Pull request closed without merging**